src/axes.jl

```diff
@@ -587,6 +588,16 @@ function round_limits(amin, amax, scale)
     amin, amax
 end
 
+parse_limits(lims::Tuple{<:Real,<:Real}) = lims
```

Comment:
Maybe better `validate_limits`? Since we don't actually do any parsing?

Reply:
It's not just validating, but also translating to a specific representation (tuple of two numbers or symbol or nothing), which is "parsing" in some general sense. Or at least, I can't think of a better word, even if "parsing" is a bit of a stretch. I don't really mind about this name either way.

Reply:
Maybe `normalize` would also fit